src/main/java/com/esri/samples/na/find_service_areas_for_multiple_facilities/README.md

```diff
@@ -0,0 +1,42 @@
+<h1>Find service areas for multiple facilities</h1>
+
+<p>Find the service areas of several facilities from a feature service.</p>
+
+<img src="FindServiceAreasForMultipleFacilities.png"/>
+
+<h2>Use case</h2>
+
+<p>A service area is a region which can be accessed from a facility as limited by one or more factors, such as travel time, distance, or cost. When analyzing the service area of multiple facilities, this workflow can be used to identify gaps in service area coverage, or significant overlaps, helping to optimize the distribution of facilities. For example, a city's health service may identify areas of a city that can be served effectively from particular hospitals, and with this optimize distribution of staff and resources.</p>
+
+<h2>How to use the sample</h2>
+
+<p>Click 'find service areas' to calculate and display the service area of each facility on the map. The polygons displayed around each facility represents the facility's service area; the red area is within 1 minute travel time from the hospital by car, whilst orange is within 3 minutes by car. All polygons are transparent, so that when service areas from multiple facilities overlap, these areas will appear as bolder colors.</p>
+
+<h2>How it works</h2>
+
+<ol>
+  <li>Create a new <code>ServiceAreaTask</code> from a network service.</li>
+  <li>Create default <code>ServiceAreaParameters</code> from the service area task.</li>
+  <li>Set the parameters <code>ServiceAreaParameters.setReturnPolygons(true)</code> to return polygons of all service areas.</li>
+  <li>Add facilities of the <code>ServiceAreaParameters</code>. For this, use a set of <code>QueryParameters</code> to select features from a <code>ServiceFeatureTable</code>: <code>serviceAreaParameters.SetFacilities(facilitiesTable, queryParameters)</code>.</li>
+  <li>Get the <code>ServiceAreaResult</code> by solving the service area task using the parameters.</li>
```

Comment:
I found the code above hard to scan, since my brain was in reading code then not English; so I didn't notice the colon separation. The two blocks of code together didn't compute. A couple of separating words would help me; 'using' might do.

Reply:
That's a very good point - I've changed the sentence to break up the code snippets.